extensions/resource-deployment/src/test/toolsService.test.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import 'mocha';
+import assert = require('assert');
+import { ToolsService } from '../services/toolsService';
+import { ToolType } from '../interfaces';
+
+suite('Tools Service Tests', function (): void {
+
+	test('run getToolByName with all known values', () => {
+		const toolsService = new ToolsService();
+
+		const tools = [['azcli', ToolType.AzCli], ['docker', ToolType.Docker], ['kubectl', ToolType.KubeCtl], ['mssqlctl', ToolType.MSSQLCtl], ['python', ToolType.Python]];
+		tools.forEach(pair => {
```

Comment:
Would be great to have this test also check that every value in the enum is defined here